bindings/c/src/common.rs

```diff
@@ -1080,9 +1085,12 @@ impl ActivationHandler for FfiActivationHandler {
 }
 
 type ActionHandlerCallbackUnwrapped =
-    extern "C" fn(request: *const action_request, userdata: *mut c_void);
+    extern "C" fn(request: *mut action_request, userdata: *mut c_void);
+
+/// Ownership of `request` is transfered to the callback. `request` must
+/// be freed using [`accesskit_action_request_free`].
```

Comment:
```suggestion
/// be freed using `accesskit_action_request_free`.
```

Remember that these doc comments only end up in the header file and not on docs.rs, so I think the brackets are only adding noise.